cmd/src/campaigns_suggestions.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"context"
+
+	"github.com/sourcegraph/src-cli/internal/api"
+	"github.com/sourcegraph/src-cli/internal/campaigns"
+	"github.com/sourcegraph/src-cli/internal/output"
+	"gopkg.in/yaml.v2"
```

Comment:
I think if you use `v3` then the `"on":` would be printed as `on:` instead.

Reply:
Now moot.